packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -23,38 +26,56 @@ function extractDomainFromEmail(email: string) {
 }
 
 export const CreateANewOrganizationForm = ({ slug }: { slug?: string }) => {
-  const { t, i18n } = useLocale();
+  const session = useSession();
+  if (!session.data) {
+    return null;
+  }
+  return <CreateANewOrganizationFormChild slug={slug} session={session} />;
+};
+
+const CreateANewOrganizationFormChild = ({
```

Comment:
Created child to ensure that useForm defaultValues are set when `useForm` is called.

Reply:
Great idea - much cleaner than having to force a re-render :P